test/functionalTest/harnessFunctions.sh

```diff
@@ -1294,7 +1294,7 @@ function orionCurl()
   then
     echo "Broker seems to have died ..."
   else
-    cat /tmp/httpHeaders.out  | sed 's/
//g' > /tmp/httpHeaders.noCtrlM
+    cat /tmp/httpHeaders.out  | sed 's/^M//g' > /tmp/httpHeaders.noCtrlM
```

Comment:
Seems like your editor is unable to deal with ASCII control characters.
I'll fix it in emacs as part of the PR.

^M is supposed to be one char, not two.

Reply:
This would be great, I was trying a lot to get it back correctly:)

Reply:
:)

Reply:
Fixed in 0b6b6e2